src/models.ts

```diff
@@ -0,0 +1,75 @@
+import { Action } from 'redux';
+
+export enum MessageTypes {
+  Error = 'ERROR',
+  Warning = 'WARNING',
+  Success = 'SUCCESS',
+  Info = 'INFO',
+  Apocalypse = 'APOCALYPSE',
+}
+
+export type ExtendableMessageTypes<T = {}> = T | MessageTypes;
```

Comment:
What is the reason behind making it extendable?

Reply:
https://github.com/42BV/redux-flash-messages/blob/5d242d7348146854d4ffaa18a08cf507fcc757b7/tests/service.test.ts#L159-L170

https://github.com/42BV/redux-flash-messages/blob/5d242d7348146854d4ffaa18a08cf507fcc757b7/README.md#L189-L205

Reply:
Ah my bad, I though you added a new functionality. But extending the message type was already part of this project.